src/CloudNative.CloudEvents.Avro/ObsoleteFormatter.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright 2021 Cloud Native Foundation. 
+// Licensed under the Apache 2.0 license.
+// See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace CloudNative.CloudEvents;
+
+/// <summary>
+/// Formatter that implements the Avro Event Format.
+/// </summary>
+/// <remarks>
+/// This class is the wrong namespace, and is only present for backward compatibility reasons.
+/// Please use CloudNative.CloudEvents.Avro.AvroEventFormatter instead
+/// (which this class derives from for convenience).
+/// </remarks>
+[Obsolete("This class is the wrong namespace, and is only present for backward compatibility reasons. Please use CloudNative.CloudEvents.Avro.AvroEventFormatter.")]
```

Comment:
I think this may be a breaking change depending on compiler options being used. Would it be sufficient to use EditorBrowsableState.Never instead?

Reply:
I've always taken the view that making something obsolete with a warning shouldn't be regarded as a breaking change - if users have explicitly opted in to making warnings errors, they're basically opting into having broken builds more often. There are any number of things which may introduce warnings but aren't normally considered as breaking changes. (Annoyingly, this isn't mentioned in the [canonical breaking change documentation](https://docs.microsoft.com/en-us/dotnet/core/compatibility/) as far as I can see.)

Reply:
Fair enough!